src/static/helpers/slasHelper.ts

```diff
@@ -118,8 +121,9 @@ export async function authorize(
     redirectURI: string;
     hint?: string;
     usid?: string;
-  },
-  privateClient = false
+  } & {[key in `c_${string}`]: any},
```

Comment:
I think we should create a type for custom query parameters and reuse it instead of using `any`:

```typescript
/**
 * Custom query parameter type with any string prefixed with `c_` as the key and the allowed 
 * types for query parameters for the value.
 */
export type CustomQueryParameters = {
  [key in `c_${string}`]: string | number | boolean | string[] | number[];
}


// in function
  parameters: {
    redirectURI: string;
    hint?: string;
    usid?: string;
  } & CustomQueryParameters,
```

It'll be a breaking change if we update it in our API files, but in the next major release we can add a note to update it to use that `CustomQueryParameters` type instead of `any` as well